pytrickle/server.py

```diff
@@ -443,22 +566,17 @@ async def _handle_update_params(self, request: web.Request) -> web.Response:
                     "message": "No active stream to update"
                 }, status=400)
             
-            # Update frame processor parameters
-            await self.frame_processor.update_params(data)
-            logger.info(f"Parameters updated: {data}")
-            
-            # Emit monitoring event
-            if self.current_client.protocol:
-                await self.current_client.protocol.emit_monitoring_event({
-                    "type": "params_updated",
-                    "timestamp": int(time.time() * 1000),
-                    "params": data
-                })
-            
-            return web.json_response({
-                "status": "success",
-                "message": "Parameters updated successfully"
-            })
+            # Schedule background update to avoid blocking the HTTP response
+            self._schedule_param_update(data)
+            logger.info(f"Parameter update scheduled: {data}")
+
+            return web.json_response(
+                {
+                    "status": "accepted",
+                    "message": "Parameter update scheduled",
```

Comment:
I would return a `uuid` here and add the `uuid` to the state when the params update is processed successfully so clients can confirm params were updated.

This can be done in a separate PR though.

Reply:
Added in https://github.com/livepeer/pytrickle/commit/b6f2e3bae33bf91e3d8dbf221d185e72e5524fc8